sematic/ui/src/pipelines/RunTree.tsx

```diff
@@ -12,12 +12,15 @@ import { Run } from "../Models";
 import RunStateChip from "../components/RunStateChip";
 import { ExtractContextType } from "../components/utils/typings";
 import PipelinePanelsContext from "./PipelinePanelsContext";
+import { HIDDEN_RUN_NAME_LIST } from "../constants";
 
 export default function RunTree(props: {
   runTreeNodes: Array<RunTreeNode>;
 }) {
   let { runTreeNodes } = props;
 
+  runTreeNodes = runTreeNodes.filter(({run}) => HIDDEN_RUN_NAME_LIST.indexOf(run?.name ?? "")===-1);
```

Comment:
Please add a code comment explaining this.

Reply:
Sure I will do it.

Reply:
I don't see `useMemo` being used here. But that's fine since it will not help much with the caching. To effectively cache things, we might have to cache the entire section of `runTreeNodes.map()`. If you want to improve, feel free to consider that. Otherwise, this looks good too.